### PR TITLE
Update  How to Use Asset Proofs guide

### PR DESCRIPTION
**Pull request by feeblefakie**

### CODE REVIEW

File: docs/how-to-use-proof.md [hunk 1]
```diff
@@ -4,28 +4,27 @@ This document sets out some guidelines for using Asset Proofs in Scalar DL.
...
-However, the ledger can tamper data from the beginning and it makes the proofs insufficient to detect the tampering.
+Since Asset Proofs are evidence at the time of execution by the Ledger, it is hard for the Ledger to tamper data after the evidence is created because the proofs and the Ledger states would be diverged.
+Thus, making use of Asset Proofs appropriately could reduce the risk of tampering of data.
 
 ## How to use Asset Proofs
 
-You can get Asset Proofs from the result of the `executeContract` method of the SDKs. Please check [the documents](https://github.com/scalar-labs/scalardl#client-sdks) of client SDKs for more detail.
-A proof can be validated if it is not tampered and it is from the ledger by verifying the signature.
+You can get Asset Proofs from the result of the `executeContract` method of the SDKs. Please check [the documents](https://github.com/scalar-labs/scalardl#client-sdks) of client SDKs for more detail. A proof can be validated if it is not tampered and it is from the Ledger by verifying the signature.
```
Comment (jnmt) [inline docs/how-to-use-proof.md:22]: Is the link to the client SDKs the following now?
https://github.com/scalar-labs/scalardl/blob/master/docs/index.md#client-sdks

File: docs/how-to-use-proof.md [hunk 2]
```diff
@@ -4,28 +4,27 @@ This document sets out some guidelines for using Asset Proofs in Scalar DL.
...
 ## What is an Asset Proof in Scalar DL?
 
-An Asset Proof in Scalar DL is a set of information about an asset record that is created along with the asset record. It is composed of the following items so that it can be treated as evidence of the creation of an asset record by a ledger.
+An Asset Proof in Scalar DL is a set of information about an asset record and used as evidence of the existence of the asset record. It is composed of the following items.
 
-- The ID of an asset record
+- ID of an asset record
 - Age of the asset record
 - Nonce of the execution request that creates the asset record
 - A cryptographic hash of the asset record
```
Comment (supl) [inline docs/how-to-use-proof.md:12]: Maybe it's not necessary but do we need to also mention the previous hash?

Comment (feeblefakie) [reply docs/how-to-use-proof.md:12]: Thank you! Fixed in [0aaea44](https://github.com/scalar-labs/scalardl/pull/31/commits/0aaea44444e1d49ea84ce6d40a74cd7703df84ae).

File: docs/how-to-use-proof.md (hunk 1 above)
Comment (feeblefakie) [reply docs/how-to-use-proof.md]: Thank you! Fixed it based on your feedback in [0aaea44](https://github.com/scalar-labs/scalardl/pull/31/commits/0aaea44444e1d49ea84ce6d40a74cd7703df84ae).

**Review by jnmt — APPROVED** [review]
Thank you for fixing. LGTM!

**Pull request merged**